### PR TITLE
Link post's author to his page

### LINKED ISSUE #34: Link post's author to his page
The post's page now shows the author's details, but we should use a `Link` component from `react-router-dom` to connect it to the proper author's details page.

sunt_aut_facere_repellat_provident_occaecati_excepturi_optio_reprehenderit

### PR DESCRIPTION
**Pull request by lenguyenitc**
close #34
link author

### CODE REVIEW

File: services/webapp/src/features/posts/containers/AuthorDetails.jsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ class AuthorDetails extends React.PureComponent {
 
     render () {
         return this.props.isReady
-            ? <Content {...this.props} />
+            ? <Link to={`/users/${this.props.authorId}/details`}><Content {...this.props} /></Link>
```
Comment (marcopeg) [inline services/webapp/src/features/posts/containers/AuthorDetails.jsx]: I think you are 90% close to getting this right.

Take a look at the definition of the "Content" component. It's few lines above.
Inside there we already have a link for the email, so this implementation ends up in a nested link that is not good.

Can you move the Link just around the author's username?

**Review by marcopeg — CHANGES_REQUESTED** [review]

File: services/webapp/src/features/posts/containers/AuthorDetails.jsx (hunk 1 above)
Comment (lenguyenitc) [reply services/webapp/src/features/posts/containers/AuthorDetails.jsx]: Thank for your feedback. I have update it

File: services/webapp/src/features/posts/containers/AuthorDetails.jsx [hunk 2]
```diff
@@ -32,11 +32,11 @@ const dispatch2props = {
 
 const Loading = () => null
 
-const Content = ({ name, email, address }) => (
+const Content = ({ id, name, email, address }) => (
     <div>
-        <PageTitle>{name}</PageTitle>
+        <PageTitle><Link to={`/users/${id}/details`}>{name}</Link></PageTitle>
```
Comment (marcopeg) [inline services/webapp/src/features/posts/containers/AuthorDetails.jsx]: can we add the style for proper color here?

File: services/webapp/src/features/posts/containers/AuthorDetails.jsx [hunk 3]
```diff
@@ -32,11 +32,11 @@ const dispatch2props = {
...
         <div>
-            <a href={`mailto:${email}`}>{email}</a>
+            <Link to={`mailto:${email}`}>{email}</Link>
```
Comment (marcopeg) [inline services/webapp/src/features/posts/containers/AuthorDetails.jsx]: this link does not work, try it out yourself.

I guess this has to be a normal anchor tag as is the browser that takes care of the `mailto:` part, so to launch the mail client.

**Review by marcopeg — CHANGES_REQUESTED** [review]

**Comment by marcopeg** [comment]
good job, thanks

**Pull request merged**